datafusion/proto/proto/datafusion.proto

```diff
@@ -1133,17 +1133,24 @@ message ScanLimit {
 
 message FileScanExecConf {
   repeated FileGroup file_groups = 1;
-  datafusion.Schema schema = 2;
+  Schema schema = 2;
   repeated uint32 projection = 4;
   ScanLimit limit = 5;
   Statistics statistics = 6;
   repeated string table_partition_cols = 7;
   string object_store_url = 8;
+  repeated PhysicalSortExprNode output_ordering = 9;
+  repeated ConfigOption options = 10;
```

Comment:
I general I am not sure it is a good idea to serialize the config options as upon deserialization they will be their own copy.

However that being said, perhaps the issue is that `FileScanExecConfig` has a copy of the config options in the first place. I think that should get better over time as we consolidate the configuration more